### ISSUE TITLE
Error when posting to: https://trinsic-mediator-agent-southafrica.azurewebsites.net/

### ISSUE THREAD

**Issue by dnaicker** [hyperledger/aries-cloudagent-python#1367]
When running the agent demo for revocation with the following steps:

1. start ngrok 8020
2. start indy-tails agent for revocation of verifiable credentials connects to separate revocation registry
3. start faber agent
4. connect via trinsic or mobile app

I receive the following errors upon start-up and upon issuing credential:

### Startup of demo agent

> Faber      | Connected                                                                                                                                                                                             Faber      | 2021-08-23 06:24:10,155 aries_cloudagent.transport.outbound.manager ERROR >>> Error when posting to: https://trinsic-mediator-agent-southafrica.azurewebsites.net/; Error: (<class 'aries_cloudagent.transport.outbound.base.OutboundTransportError'>, OutboundTransportError('Unexpected response status 500, caused by: Internal Server Error',), <traceback object at 0x7fb2a68544c8>); Re-queue failed message ...   Faber      | 2021-08-23 06:24:20,445 aries_cloudagent.transport.outbound.manager ERROR >>> Error when posting to: https://trinsic-mediator-agent-southafrica.azurewebsites.net/; Error: (<class 'aries_cloudagent.transport.outbound.base.OutboundTransportError'>, OutboundTransportError('Unexpected response status 500, caused by: Internal Server Error',), <traceback object at 0x7fb2a6854808>); Re-queue failed message ...   Faber      | 2021-08-23 06:24:30,539 aries_cloudagent.transport.outbound.manager ERROR >>> Error when posting to: https://trinsic-mediator-agent-southafrica.azurewebsites.net/; Error: (<class 'aries_cloudagent.transport.outbound.base.OutboundTransportError'>, OutboundTransportError('Unexpected response status 500, caused by: Internal Server Error',), <traceback object at 0x7fb2a6844748>); Re-queue failed message ...     

![image](https://user-images.githubusercontent.com/7840952/130400995-e9190668-b94e-4c1f-a4d4-11dcf6292aed.png)

### Issuing credential error after scanning QR code
> Faber      | 2021-08-23 06:24:41,333 aries_cloudagent.transport.outbound.manager ERROR >>> Error when posting to: https://trinsic-mediator-agent-southafrica.azurewebsites.net/; Error: (<class 'aries_cloudagent.transport.outbound.base.OutboundTransportError'>, OutboundTransportError('Unexpected response status 500, caused by: Internal Server Error',), <traceback object at 0x7fb2a47e0508>); Re-queue failed message ...   Faber      | Credential: state = offer_sent, credential_exchange_id = 819439fb-01a7-44f7-ac00-82c24f92e09d                                                                                                         Faber      | 2021-08-23 06:24:50,740 aries_cloudagent.transport.outbound.manager ERROR >>> Outbound message failed to deliver, NOT Re-queued.                                                                      Faber      | Traceback (most recent call last):                                                                                                                                                                    Faber      |   File "/home/indy/aries_cloudagent/transport/outbound/http.py", line 86, in handle_message                                                                                                           Faber      |     f"Unexpected response status {response.status}, "                                                                                                                                                 Faber      | aries_cloudagent.transport.outbound.base.OutboundTransportError: Unexpected response status 500, caused by: Internal Server Error                                                                     Faber      | 2021-08-23 06:24:50,785 aries_cloudagent.transport.outbound.manager ERROR Outbound message could not be delivered to https://trinsic-mediator-agent-southafrica.azurewebsites.net/                    Faber      | Traceback (most recent call last):                                                                                                                                                                    Faber      |   File "/home/indy/aries_cloudagent/transport/outbound/http.py", line 86, in handle_message                                                                                                           Faber      |     f"Unexpected response status {response.status}, "                                                                                                                                                 Faber      | aries_cloudagent.transport.outbound.base.OutboundTransportError: Unexpected response status 500, caused by: Internal Server Error                                                                     Faber      | 2021-08-23 06:24:51,393 aries_cloudagent.transport.outbound.manager ERROR >>> Error when posting to: https://trinsic-mediator-agent-southafrica.azurewebsites.net/; Error: (<class 'aries_cloudagent.transport.outbound.base.OutboundTransportError'>, OutboundTransportError('Unexpected response status 500, caused by: Internal Server Error',), <traceback object at 0x7fb2a4801088>); Re-queue failed message ...   Faber      | 2021-08-23 06:25:01,472 aries_cloudagent.transport.outbound.manager ERROR >>> Error when posting to: https://trinsic-mediator-agent-southafrica.azurewebsites.net/; Error: (<class 'aries_cloudagent.transport.outbound.base.OutboundTransportError'>, OutboundTransportError('Unexpected response status 500, caused by: Internal Server Error',), <traceback object at 0x7fb2a68544c8>); Re-queue failed message ...   Faber      | 2021-08-23 06:25:11,568 aries_cloudagent.transport.outbound.manager ERROR >>> Error when posting to: https://trinsic-mediator-agent-southafrica.azurewebsites.net/; Error: (<class 'aries_cloudagent.transport.outbound.base.OutboundTransportError'>, OutboundTransportError('Unexpected response status 500, caused by: Internal Server Error',), <traceback object at 0x7fb2a6854948>); Re-queue failed message ...   Faber      | 2021-08-23 06:25:21,652 aries_cloudagent.transport.outbound.manager ERROR >>> Outbound message failed to deliver, NOT Re-queued.                                                                      Faber      | Traceback (most recent call last):                                                                                                                                                                    Faber      |   File "/home/indy/aries_cloudagent/transport/outbound/http.py", line 86, in handle_message                                                                                                           Faber      |     f"Unexpected response status {response.status}, "                                                                                                                                                 Faber      | aries_cloudagent.transport.outbound.base.OutboundTransportError: Unexpected response status 500, caused by: Internal Server Error                                                                     Faber      | 2021-08-23 06:25:21,653 aries_cloudagent.transport.outbound.manager ERROR Outbound message could not be delivered to https://trinsic-mediator-agent-southafrica.azurewebsites.net/                    Faber      | Traceback (most recent call last):                                                                                                                                                                    Faber      |   File "/home/indy/aries_cloudagent/transport/outbound/http.py", line 86, in handle_message                                                                                                           Faber      |     f"Unexpected response status {response.status}, "                                                                                                                                                 Faber      | aries_cloudagent.transport.outbound.base.OutboundTransportError: Unexpected response status 500, caused by: Internal Server Error   

![image](https://user-images.githubusercontent.com/7840952/130400449-f0979cf3-5629-4d0f-abba-ee24a034cb8f.png)

**Comment by swcurran** [comment]
@ianco  -- can you please take a quick look at this?  Thanks

**Comment by ianco** [comment]
Is this still an issue?  It looks like the error is being generated by the Trinsic endpoint  https://trinsic-mediator-agent-southafrica.azurewebsites.net/

(The error `Unexpected response status 500, caused by: Internal Server Error` means that this is the response aca-py is receiving from the remote agent.)

**Comment by dnaicker** [comment]
Its been a while since I had this setup. I am not sure if it is still returning this error.  I tried to get a qr code generated but not using the initial steps outlined, but using the other demo steps for aca-py agent, to see if the error still persists, I started using bcovrin instead of sovrin, unsure which exact tutorial documentation I was following, it was related to revocations as a special use case. The qr code was not able to be processed on the trinsic wallet app using bcovrin test network. That is probably to be expected because I was testing without following initial documentation (can't find, bit exhausted) to try and see if the error persisted.

Would someone be able to check if the azure server is still operational?
